Sources/Plasma/FeatureLib/pfConsole/pfConsoleDirSrc.cpp

```diff
@@ -61,13 +61,13 @@ bool pfConsoleDirSrc::ParseDirectory(const plFileName& path, const char* mask /*
     hsAssert(fEngine != nullptr, "Cannot do a dir execute without an engine!");
 
     std::vector<plFileName> files = plFileSystem::ListDir(path, mask);
-    for (auto iter = files.begin(); iter != files.end(); ++iter)
+    for (auto& file : files)
     {
-        plFileName name = iter->GetFileName();
+        plFileName name = file.GetFileName();
```

Comment:
`file` is already a `plFileName`. `GetFileName()` returns an `ST::string`, so we are making an unnecessary copy here.